conf.ini

```diff
@@ -0,0 +1,3 @@
+[DEFAULT]
+database = sqlite:////tmp/kostyor.db
```

Comment:
Database often include bunch of other options to configure such as connection pool, for example. So I'd prefer to have the `database` section with `connection` option in order to easily extend it if needed.

```
[database]
conection = sqlite:///tmp/kostyor.db
```

Does it make sense?